core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/MigrationUtils.scala

```diff
@@ -57,6 +65,37 @@ object MigrationUtils {
     implicit class FutureToIO[+A](future: => Future[A]) {
       def io: IO[A] = IO.fromFuture(IO(future))
     }
+
+    // TODO: move into workbench-libs
+    implicit class StorageServiceExtensions[F[_]](storageService: GoogleStorageService[F]) {
+      final def policyToStorageRoles(policy: Policy): Map[StorageRole, NonEmptyList[Identity]] =
+        policy
+          .getBindings
+          .foldLeft(Map.newBuilder[StorageRole, NonEmptyList[Identity]]) { (builder, binding) =>
+            NonEmptyList.fromList(binding._2.toList).map { identities =>
+              builder += (StorageRole.CustomStorageRole(binding._1.getValue).asInstanceOf[StorageRole] -> identities)
```

Comment:
`StorageRole`s are just a wrapper around a string. The `StorageRoles` object contains some pre-defined wappers (like `ObjectViewer`) as we as a `CustomStorageRole` constructor that's effectively "whatever's not listed here". Instead of matching on the string and returning the relevant wrapper, I'm just creating a `CustomStorageRole` because this is being unpacked straightaway in the `GoogleStorageService` implementation.

Reply:
Comment added.